store/src/test/java/org/apache/rocketmq/store/DefaultMessageStoreTest.java

```diff
@@ -263,7 +270,7 @@ public void should_return_zero_when_consume_queue_not_found() throws Interrupted
     }
 
     @Test
-    public void should_return_negative_one_when_invoke_getMessageStoreTimeStamp_if_consume_queue_not_found() throws InterruptedException {
+    public void should_return_negative_one_when_invoke_getMessageStoreTimeStamp_if_consume_queue_not_found() throws InterruptedException, UnknownHostException {
```

Comment:
Can you modify the test method signature of this section according to code style(test[method]_[conditon] ) in Apache RocketMQ ?

Reply:
Thank you for your opinion. It has been modified as required.